DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1070 @@ Live discussion in #document-authoring on Slack.
+
+The `AdaptationSet@subsegmentStartsWithSAP` attribute SHALL be present in the MPD and SHALL have a value of `1` or `2`, depending on the sample structure of the [=media segments=].
+
+Issue: We need to clarify how to determine the right value. [#235](https://github.com/Dash-Industry-Forum/DASH-IF-IOP/issues/235)
+
+The `SegmentBase/Initialization@range` attribute SHALL reference the byte range of the initialization segment in the track file. The value SHALL be formatted as a `byte-range-spec` as defined in [[!RFC7233]], referencing a single range of bytes. The `Initialization@sourceURL` attribute SHALL NOT be used.
+
+<div class="example">
+Below is an example of common usage of the indexed addressing mode.
+
+The example defines a timescale of 48000 units per second, with the period starting at position 8100 (or 0.16875 seconds) on the [=sample timeline=]. The client can use the index segment referenced by `indexRange` to determine where the [=media segment=] containing position 8100 (and all other [=media segments=]) can be found. The byte range of the initialization segment is also referenced.
+
+<xmp highlight="xml">
+<MPD xmlns="urn:mpeg:dash:schema:mpd:2011">
+	<Period>
+		<AdaptationSet subsegmentStartsWithSAP="1">
+			<Representation>
+				<BaseURL>showreel_audio_dashinit.mp4</BaseURL>
+				<SegmentBase timescale="48000" presentationTimeOffset="8100" indexRange="848-999">
+					<Initialization range="0-847"/>
+				</SegmentBase>
+			</Representation>
+		</AdaptationSet>
+	</Period>
+</MPD>
+</xmp>
+
+Parts of the MPD structure that are not relevant for this chapter have been omitted - this is not a fully functional MPD file.
+</div>
+
+#### Structure of the index segment #### {#timing-addressing-indexed-indexstructure}
+
+The index segment SHALL consist of a single Segment Index Box (`sidx`) as defined by [[!ISOBMFF]]. The field layout is as follows:
+
+<xmp>
+aligned(8) class SegmentIndexBox extends FullBox('sidx', version, 0) {
+	unsigned int(32) reference_ID;
+	unsigned int(32) timescale;
+
+	if (version==0) {
+		unsigned int(32) earliest_presentation_time;
+		unsigned int(32) first_offset;
+	}
+	else {
+		unsigned int(64) earliest_presentation_time;
+		unsigned int(64) first_offset;
+	}
+
+	unsigned int(16) reserved = 0;
+	unsigned int(16) reference_count;
+
+	for (i = 1; i <= reference_count; i++)
+	{
+		bit (1)              reference_type;
+		unsigned int(31)     referenced_size;
+		unsigned int(32)     subsegment_duration;
+		bit(1)               starts_with_SAP;
+		unsigned int(3)      SAP_type;
+		unsigned int(28)     SAP_delta_time;
+	}
+}
+</xmp>
+
+The values of the fields SHALL be as follows:
+
+: `reference_ID`
+:: The `track_ID` of the track that contains the data of this [=representation=].
+
+: `timescale`
+:: Same as the `timescale` field of the Media Header Box and same as the `SegmentBase@timescale` attribute in the MPD.
```

Comment:
From @haudiobe: Match the time scale of the track header would be much better, but why is this important?

Reply:
We say in IOP that SegmentTimeline timescale must match ISOBMFF timescale. The idea here is specifically to link the MP4 and the BMFF values. I just put the existing IOP logic into words here without deeper thought but I suppose it would not be too hard to think of situations where using different timescale on BMFF and MPD level could cause client implementation issues.